app/mailers/output_document_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class OutputDocumentMailer < ActionMailer::Base
+  def issue(appointment_summary, output_document)
```

Comment:
The `appointment_summary` argument implies a dependency on an object of type `AppointmentSummary`.

You could name it something that implies duck typing, e.g. `recipient_details` though given we only need an email address, just have that as the param?